config/database.php

```diff
@@ -130,3 +130,11 @@
     ],
 
 ];
+
```

Comment:
Can you add documentation in the code for this, as comments? I just don't want that someone reading this won't understand why this is necessary in some cases. Thank you!